cylc/flow/network/__init__.py

```diff
@@ -46,19 +45,17 @@
 MSG_TIMEOUT = "TIMEOUT"
 
 
-def encode_(message):
+def JSON_stringify(message: object) -> str:
     """Convert the structure holding a message field from JSON to a string."""
     try:
         return json.dumps(message)
     except TypeError as exc:
         return json.dumps({'errors': [{'message': str(exc)}]})
 
 
-def decode_(message):
-    """Convert an encoded message string to JSON with an added 'user' field."""
-    msg = json.loads(message)
-    msg['user'] = getpass.getuser()  # assume this is the user
-    return msg
+def JSON_parse(message: str) -> object:
```

Comment:
Same here.. You'd be hard pressed to find any other mixed case functions in our code.